modules/simple_forms_api/spec/services/confirmation_email_spec.rb

```diff
@@ -203,4 +203,37 @@
       end
     end
   end
+
+  describe '20_10206' do
+    let(:data) do
+      fixture_path = Rails.root.join(
+        'modules', 'simple_forms_api', 'spec', 'fixtures', 'form_json', 'vba_20_10206.json'
+      )
+      JSON.parse(fixture_path.read)
+    end
+
+    describe 'citizen or non-citizen' do
```

Comment:
This is totally a nit but do we need this `describe` block? I don't see any other `describe` blocks that present an alternative path so this should work with the `it` block below coming directly after we define `:data`, right?

Reply:
Yes! I had separate blocks for citizen/non-citizen; but when those were deemed unnecessary, I didn't remove the `describe`